server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java

```diff
@@ -0,0 +1,343 @@
+package io.unitycatalog.server.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.unitycatalog.server.model.*;
+import io.unitycatalog.server.persist.CatalogRepository;
+import io.unitycatalog.server.persist.FunctionRepository;
+import io.unitycatalog.server.persist.HibernateUtil;
+import io.unitycatalog.server.persist.SchemaRepository;
+import io.unitycatalog.server.persist.dao.*;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+
+import java.util.*;
+
+import static io.unitycatalog.server.utils.ColumnUtils.*;
+
+/**
+ * This is a utility class to populate the test database with some sample data.
+ * All the quickstart examples in the documentation are based on this sample data.
+ * This class is not part of the main application code and is only used for testing purposes.
+ * To recreate the sample data, first delete the existing database (at /etc/db)
+ * and then run main method of this class by running the command `build/sbt server/populateTestDB`.
```

Comment:
its `populateTestDatabase` isnt it?

Reply:
This is just the name of the target specified in the `build.sbt`.